docs/installation.rst

```diff
@@ -0,0 +1,149 @@
+  pip install unidist[mpi] # Install unidist with dependencies for MPI backend
+  pip install unidist[ray] # Install unidist with dependencies for Multiprocessing and sequential Python backends
+
+unidist automatically detects which execution backends are installed and use that for
+scheduling computation!
+
+Release candidates
+""""""""""""""""""
+
+unidist follows `Semantic Versioning`_ and before some major or minor releases,
+we will upload a release candidate to test and check if there are any problems.
+If you would like to install a pre-release of unidist, run the following:
+
+.. code-block:: bash
+
+  pip install --pre unidist
+
+These pre-releases are uploaded for dependencies and users to test their existing code
+to ensure that it still works. If you find something wrong, please raise an issue_.
+
+Installing with conda
+'''''''''''''''''''''
+
+Using conda-forge channel
+"""""""""""""""""""""""""
+
+unidist releases can be installed using ``conda`` from the conda-forge channel. Starting from the first 0.1.0 release
+it is possible to install unidist with chosen execution backend(s) alongside. Current options are:
+
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| **Package name in conda-forge** | **Backend(s)**                                      | **Supported OSs**           |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist                         | `Multiprocessing`_, Python                          | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-all                     | `Dask`_, `MPI`_, `Ray`_, `Multiprocessing`_, Python | Linux, Windows              |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-dask                    | `Dask`_                                             | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-mpi                     | `MPI`_                                              | Linux, Windows, MacOS       |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+| unidist-ray                     | `Ray`_                                              | Linux, Windows              |
++---------------------------------+-----------------------------------------------------+-----------------------------+
+
+Installing unidist packages from the conda-forge channel can be achieved by adding conda-forge to your channels with:
+
+.. code-block:: bash
+
+  conda config --add channels conda-forge
+  conda config --set channel_priority strict
```

Comment:
I'm not sure that this is needed actions. We already specify channel `conda-forge` during installation. Why do I need to change my conda config?

Reply:
Some inattentive users may miss `-c conda-forge` in the installation line so I think we should be explicit.

Reply:
ok, let's remove for now